cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -37,46 +37,33 @@
 
 #include "cob_obstacle_distance/Registration.h"
 
-#define DEBUG_BASE_ACTIVE    0
-#define DEBUG_BASE_COMP     0
+#define MESH_RES_MARKER 99u // Must be equal to "cob_obstacle_distance/obstacle_distance_data_types.hpp"
```

Comment:
Correct... Removed define and added MESH_RESOURCE from visualization marker (did not see that 8-D)